sunpy/map/mapbase.py

```diff
@@ -2730,7 +2730,8 @@ def contour(self, level, **kwargs):
             # find_contours expects a scalar below
             level = level[0]
 
-        contours = measure.find_contours(self.data, level=level, **kwargs)
+        gen = contour_generator(z=self.data, **kwargs)
```

Comment:
How much do we care about this method using the same algorithm as `draw_contour`? It seems that there are faster options in contourpy (like the threaded one) we could default to?

Reply:
In theory now, they should be the same as MPL uses it.

Reply:
yeah, but my point is that there is apparently a newer algorithm in contourpy not used by mpl (yet) which is faster and can be multi-threaded. https://contourpy.readthedocs.io/en/v1.2.1/user_guide/calculate/threads.html#threads

Reply:
Lets use that.